MabiCommerce/data/db/transports.json

```diff
@@ -31,5 +31,29 @@
     "Slots": 7,
     "Weight": 1700,
     "Id": 3
-  }
+  },
+  {
+    "Name": "Dog Sled",
+    "Icon": "data/img/transport/elephant.png",
+    "SpeedFactor": 1.88,
```

Comment:
If I recall correctly, the speeds are in one of the xml files called DataTransports or something similar

Reply:
Ok cool, I'll see if I can find it, do you think its a bare file some where? or would i need to extract it from a "pack"?

Reply:
Ah, I see many pack files, .its.
Was there a list of which packs contain what or was it just, unpack everything and look around?

Reply:
Back in the day, it was just a bunch of files with .pack extensions. Seems like they've changed since I last dealt with them, so unfortunately I'm going to be of limited help here, I fear.

That said, in absence of any discernable naming scheme, I'd just unpack them all.